### ISSUE TITLE
Support air-gapped tx signing over sd-card

### ISSUE THREAD

**Issue by slush0** [trezor/trezor-firmware#72]
Trezor will read unsigned transaction template from sd-card, let user confirm the transaction on the display and put signed transaction back to the card. This will let people use trezor in completely air-gapped environment as powering the device through power bank is enough.

**Comment by prusnak** [comment]
We need to decide which filesystem to use first: https://github.com/trezor/trezor-firmware/issues/162

**Comment by ZdenekSL** [comment]
These issues have to be done before we can work on this:
https://github.com/trezor/trezor-firmware/issues/49
https://github.com/trezor/trezor-firmware/issues?utf8=%E2%9C%93&q=project%3Atrezor%2F10+

**Comment by ZdenekSL** [comment]
Closed as duplicate of #49
